lib/RuntimeClient/index.ts

```diff
@@ -94,6 +104,26 @@ export class RuntimeClient<V extends Record<string, any> = Record<string, any>>
     throw new VFTypeError(`event "${event}" is not valid`);
   }
 
+  async onResponse(responseHandlerSideEffect: ResponseHandlerSideEffect<V>) {
+    this.responseHandlerSideEffect = responseHandlerSideEffect;
+  }
+
+  async responseHandler() {
```

Comment:
can we call this function `buildResponse()` and rename `responseHandlerSideEffect` to `responseHandler`?

They technically both handle the response, but I guess we don't need to define our internal functions as `handlers`, it is on the same level as `sendRequest` so it seems natural to do something similar